packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -170,6 +172,11 @@ class PaginatedDataTable extends StatefulWidget {
   /// [PaginatedDataTable] constructor is called.
   final DataTableSource source;
 
+  /// If set to [DragStartBehavior.start], scroll drag behavior will
+  /// begin upon the drag gesture winning the arena. If set to
+  /// [DragStartBehavior.down] it will begin when a down event is first detected.
```

Comment:
you have this paragraph in a bunch of places and sometimes you use @macro and sometimes not, which seems inconsistent.

Reply:
Sometimes I change the content of this paragraph slightly if it's not the widget scroll behavior that's affected. But for the others, I will change to use the macro.